docs/ml-guide.md

```diff
@@ -157,6 +174,49 @@ There are now several algorithms in the Pipelines API which are not in the lower
 * [Feature Extraction, Transformation, and Selection](ml-features.html)
 * [Ensembles](ml-ensembles.html)
 
+## Linear Methods with Elastic Net Regularization
+
+In MLlib, we implement popular linear methods such as logistic regression and linear least squares with L1 or L2 regularization. Refer to [the linear methods section](mllib-linear-methods.html) for details. In `spark.ml`, we add the [Elastic net](http://users.stat.umn.edu/~zouxx019/Papers/elasticnet.pdf), which is a hybrid of L1 and L2 regularization. Mathematically it is defined as a linear combination of the L1-norm and the L2-norm:
```

Comment:
> In `spark.ml`, we add the [Elastic net](http://users.stat.umn.edu/~zouxx019/Papers/elasticnet.pdf), which is a hybrid of L1 and L2 regularization.

This makes it sound like it's only in spark.ml.  Can this please instead say that we provide a Pipelines API?  The main thing needed here is a code example, which should demonstrate how to do L1, L2, and a mix.  (But I like the note about how it uses a different optimizer.)